### DISCUSSION TITLE
Advanced features for cds-layout

### DISCUSSION THREAD

**Discussion by mcosta74** [vmware/clarity#5443] [Ideas] [answered]
HI all,

I started to play with `cds-layout` and it's super-cool.
I'd like to ask if there're plans to introduce 2 features (available in pure `flex` layouts):

- reverse layout (very useful in case of RTL support)
- item order (useful in case of responsive design)

If I could give a priority I'd say the first one is more important but ... I'm just asking if there're plans.

Does it make sense to open a Feature Request r this is going to be refused?

Thanks a lot for your JOB

**Comment by mcosta74** [comment, marked as answer]
Just found that there's `order:reverse` property for `cds-layout` and .... it's what I meant. It's just not documented yet

Here an [example](https://stackblitz.com/edit/cds-dynamic-layout)

**Comment by hippee-lee** [comment]
@mcosta74 - I've accepted your answer for the reverse layout part of the question. Thanks for the follow-up.

Wrt - item order ... I'm not sure if we will be able to add it. There are [accessibility issues](https://drafts.csswg.org/css-grid/#order-accessibility) when changing only the visual order of items.

**Reply by mcosta74** [reply]
Ok, thanks for the reply

**Reply by coryrylan** [reply]
The reverse util should be used with caution as for the a11y reasons @hippee-lee pointed out. Its primary use case is to help with RTL lang support. Outside of that I would be careful that changing order does not break a11y.
